HelloAPI-server/Controllers/ValuesController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+
+namespace HelloAPI_server.Controllers
+{
```

Comment:
What is this class for?

Reply:
Deleted. It was created as part of the File - New - Project template.